src/GraphQL/Types/Collections/SchemaTypes.cs

```diff
@@ -270,7 +270,7 @@ private static IEnumerable<IGraphType> GetSchemaTypes(ISchema schema, IServicePr
         /// </summary>
         public int Count => Dictionary.Count;
 
-        private IGraphType BuildNamedType(Type type, Func<Type, IGraphType> resolver) => type.BuildNamedType(t => FindGraphType(t) ?? resolver(t));
+        private IGraphType BuildNamedType(Type type, Func<Type, IGraphType> resolver) => type.BuildGraphQLType(t => FindGraphType(t) ?? resolver(t));
```

Comment:
This is the only place the extension method is used.  How about making it a `protected virtual` method here and removing the extension method?

Reply:
done